packages/guides-code/src/Code/Highlighter/Highlighter.php

```diff
@@ -47,13 +47,6 @@ public function __invoke(string $language, string $code): HighlightResult
             $highlightValue = $highlight->value;
             $highlightLanguage = $highlight->language;
 
-            if ($language === 'terminal') {
```

Comment:
I see we have tests for this behavior. To me, this is very specific to how we want to show terminal blocks in the Symfony docs (so I'm happy to keep it in our project). Or do you think this is generic enough to keep in this project?

Reply:
it makes sense to me to remove this, and move this to your own project, this seems to be very specific indeed.